packages/smart/types/index.ts

```diff
@@ -1,13 +0,0 @@
-import { ContractDeployConfig, EtherscanVerificationConfig } from "../src";
```

Comment:
@justinbarry I want to keep the hardhat config type changes with their tasks because that's how hardhat plugins work. It's just less conceptual overhead. They're also totally internal types ,despite overriding hardhat's type) so we don't want to export them from the `smart` package. If there's a compelling reason to consolidate them then I'm cool with that.

Reply:
I figured that's why you moved the types around: to make it easy to export them for the config typing. If not then nevermind.